### PR TITLE
Don't let google users add custom XYZ/TMZ/... basemaps

### PR DESCRIPTION
**Pull request by xavijam**
It comes from this merged [PR](https://github.com/CartoDB/cartodb/pull/6334) and then [reverted](https://github.com/CartoDB/cartodb/pull/6359) due to color was disabled too.

---

Standard user:
screen shot 2016-01-11 at 16 05 42

screen shot 2016-01-12 at 11 58 06

---
Google user (_*don't take into account small previews, they are wrong_):
screen shot 2016-01-11 at 16 05 55

screen shot 2016-01-12 at 11 59 37

---

cc @yasharora73 @javisantana @saleiva

### CODE REVIEW

**Comment by xavijam** [comment]
@javierarce same you reviewed in https://github.com/CartoDB/cartodb/pull/6334, but now custom color/img is still available for any user type, could you take a look in any case?

File: lib/assets/javascripts/cartodb/table/basemap/basemap_dropdown/color_basemap.js [hunk 1]
```diff
@@ -60,11 +60,11 @@ cdb.admin.BackgroundMapColorView = cdb.admin.BaseLayerButton.extend({
     this.color_picker = new cdb.admin.ColorPicker({
       className: 'dropdown color_picker basemap border vertical_offset',
       target: this.$el,
-      vertical_position: "down",
+      vertical_position: "up",
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/basemap/basemap_dropdown/color_basemap.js]: Just moving the position of the basemap color picker.

Comment (javierarce) [inline lib/assets/javascripts/cartodb/table/basemap/basemap_dropdown/color_basemap.js]: When you say 'is still available for any user type' … you mean that's the expected behaviour or is a bug?

Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/basemap/basemap_dropdown/color_basemap.js]: Set a colored basemap should be only enabled for standard users (no for google ones) but seems like it is being used, so we will leave as it is. We will manage that non-desired feature in the new editor, I guess.

**Comment by javierarce** [comment]
:+1: Code looks good to me.

File: lib/assets/javascripts/cartodb/table/basemap/basemap_dropdown/color_basemap.js (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/table/basemap/basemap_dropdown/color_basemap.js]: :ok_hand:

**Pull request merged**